src/headless/HeadlessApps.h

```diff
@@ -0,0 +1,23 @@
+#include "SurgeSynthesizer.h"
+
+namespace Surge
+{
+namespace Headless
+{
+
+/**
+ * runSimpleSynthToStdout
+ *
+ * Shows how to create and run a synth putting output to stdout
+ */
+void runSimpleSynthToStdout(SurgeSynthesizer* surge);
+
+/**
+ * scanAllPresets
+ *
+ * Loads each preset, scans for volume characteristics under a few playing modes
+ */
+void scanAllPresets(SurgeSynthesizer* surge);
```

Comment:
```
void playNote(SurgeSynthesize* surge);

void playNoteOnAllPresets(SurgeSynthesize* surge);
```

We can always refactor the API later on more generic but these exactly describe what is going on right now.

Reply:
Yeah I will do a bit more expansive a factor now that I see what you see in the current code. Very very useful as always thank you!